README.md

```diff
@@ -1 +1,64 @@
-# bpmn-js-tracking
\ No newline at end of file
+# bpmn-js-tracking
+
+An extension for [bpmn-js](https://github.com/bpmn-io/bpmn-js) to track user interaction.
+
+
+## Installation
+
+Install via [npm](http://npmjs.com/).
+
+```
+npm install bpmn-js-tracking
+```
+
+
+## Usage
+
+Add as additional module to [bpmn-js](https://github.com/bpmn-io/bpmn-js).
+
+```javascript
+import BpmnJSTracking from 'bpmn-js-tracking';
+
+const bpmnJS = new BpmnJS({
+  additionalModules: [
+    BpmnJSTracking
+  ],
+  bpmnJSTracking: trackingConfiguration
```

Comment:
What is the `trackingConfiguration`?

Why document two things (both events _and_ static configuration)? I'd rather only expose one option.

Reply:
What would be the purpose of passing `optIn` and `optOut`. 

I understand the purpose of the events; allowing other components to contribute behavior. I don't get the purpose of these `opt*` callbacks.

----

Generally what helps is to separate internal (extension points) from external (to be used by consumers) APIs.

Let's settle down on:

* Externally you'd pass the callback => We document it
* Internally there may be events thrown (for the purpose to hook in) => Document it in a dedicated, separate section

Hope this makes sense.